ui/library/less/styles.less

```diff
@@ -20,21 +11,44 @@
 
 .reader__page-overlay__bounding-box {
   position: absolute;
-  z-index: @z-index-bbox-overlay;
+  z-index: @z-index-boundingbox-overlay;
```

Comment:
Nit: `bounding-box` instead of `boundingbox` for consistency

Reply:
cool will fix it